R/odbc-connection.R

```diff
@@ -176,21 +180,33 @@ NULL
 #'
 #' @rdname odbcConnectionColumns
```

Comment:
I think you could `@keywords internal` and make sure it's not include in the pkgdown ref too.

Reply:
Sounds good! 5096a4b.